CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.7.12
+
+* Fix kubectl permissions.
```

Comment:
@tsu-shiuan you missed this copy-🍝 ... fixed in https://github.com/Intellection/docker-kubectl/pull/11/commits/1e7a8f2e4c82d14f56c993a2456274112e5762e3.